djangocms_navigation/cms_menus.py

```diff
@@ -17,7 +19,8 @@ def get_roots(self, request):
         )
         versionable = get_versionable_for_content(MenuContent)
         if versionable:
-            menucontents = versionable.distinct_groupers()
+            inner_filter = {"versions__state__in": [DRAFT, PUBLISHED]}
```

Comment:
Doesn't this mean that both DRAFT and PUBLISHED versions will always appear? And really what we want I guess is for both to appear in preview but only published ones to appear on the site, right?